assistant_dists/dream_ranking_and_sfc_based_dm/docker-compose.override.yml

```diff
@@ -0,0 +1,444 @@
+services:
+  agent:
+    command: sh -c 'bin/wait && python -m deeppavlov_agent.run agent.channel=telegram agent.telegram_token=$TG_TOKEN agent.pipeline_config=assistant_dists/dream_Speech_models_based/pipeline_conf.json agent.db_config=assistant_dists/dream_Speech_models_based/db_conf.json'
+    environment:
+      WAIT_HOSTS: "sentseg:8011, ranking-based-response-selector:8002,
+          dff-intent-responder-skill:8012, intent-catcher:8014, ner:8021,
+          factoid-qa:8071, kbqa:8072, entity-linking:8075, wiki-parser:8077, text-qa:8078,
+          combined-classification:8087, fact-retrieval:8100, entity-detection:8103,
+          speech-function-classifier:8108,
+          sentence-ranker:8128, property-extraction:8136, prompt-selector:8135, openai-api-chatgpt:8145,
+          openai-api-chatgpt-16k:8167, summarization-annotator:8058, dialog-summarizer:8059"
```

Comment:
why did you remove ALL llm-based skills?

Reply:
I do not completely understand - where is a new response selector? you had to create a new response selector, and make a function to integrate SF to response selection

Reply:
That's a good question. The removal was accidental and unintentional, we're taking them back rn

Reply:
- new response selector has arrived 
- regarding llm-based skills, we had to drop `dff-dream-faq-prompted-skill`, `dff-google-api-skill` and `dff-dream-persona-chatgpt-prompted-skill` bc they require api keys which we didnt have atm. If the skills are crucial, we'll add them back when we know what to add to .env_secret file